docs/tasks/semester-checklist.rst

```diff
@@ -1,9 +1,39 @@
 Semester Checklist
 ==================
-
 A quick checklist of tasks that need to be completed at the start of every semester.
 
--  Reserve the meeting room by speaking to the lab manager (requires a signature from the president and vice president). Get set up with swipe access if need be.
+Before Semester
+---------------
+Tasks to do during the break.
+
+-  Hold weekly/bi-weekly eboard meetings.
+-  Discuss plans and brainstorm for next semester.
+-  Reserve meeting space.
+-  Triage and review tasks from previous semester to plan for coming semester (see project board).
+
+Before Fall Semester
+----------------------
+Tasks which only need to be completed before the fall semester.
+
+-  Get all eboard members financially certified through Campus Life.
+-  Plan for the Activities Gala.
 -  Complete any outstanding eboard :doc:`transition tasks <../administration/eboard-onboarding-offboarding>`
--  Set up a meeting with the club faculty adviser to discuss plans for the semester
--  Update the website and thelink for the new semester
+
+At Start of Semester
+--------------------
+Tasks for last few weeks of break.
+
+-  Set up a meeting with the club faculty adviser to discuss plans for the semester.
+-  Create a poster for the start of the semester.
+-  Present at Activities Gala (fall semester only).
```

Comment:
You can optionally do a cross-document link here too, if you'd like (per previous instructions).